renovate.json

```
@@ -7,7 +7,7 @@
   "ignorePaths": [".yarn"],
   "postUpdateOptions": ["yarnDedupeHighest"],
   "lockFileMaintenance": { "enabled": true },
-  "ignoreDeps": ["node", "cimg/node", "kill-port", "semantic-release"],
+  "ignoreDeps": ["node", "cimg/node", "kill-port", "semantic-release", "cms"],
```

[boulder] This is adding Decap to the list of ignored dependencies, correct?

I don't think we want to entirely skip updates. Could we instead add another entry to `packageRules` below? We could set `automerge: false` and this would still allow the PR to get created, but we could elect to include the update or not.

---

I understand that the current version is giving us some grief, but assuming that a future patch fixes our issues this might be a "softer" fix to prevent breaking upgrades that maintains visibility.

---

Added cms below in the renovate.json

---

[boulder] If we have a rule to prevent auto-merge, we don't need this.
```suggestion
  "ignoreDeps": ["node", "cimg/node", "kill-port", "semantic-release"],
```

---

decap-cms-app is the correct package. we will update it.